### PR TITLE
Minor code improvements in CachingJdbcClient

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

**Review by kokosing — APPROVED** [review]

**Review by ssheikin — APPROVED** [review]

**Comment by kokosing** [comment]
Please capture failure from logs as github issue to capture flakiness.

File: presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java [hunk 1]
```diff
@@ -14,7 +14,6 @@
 package io.prestosql.plugin.jdbc;
 
 import com.google.common.annotations.VisibleForTesting;
-import com.google.common.base.Predicate;
```
[the remark this answers is not in the document]
Comment (wendigo) [reply presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java:17]: Makes sense

**Comment by kokosing** [comment]
Flaky failure: https://github.com/prestosql/presto/issues/5758

**Comment by kokosing** [comment]
Merged, thanks!

**Pull request merged**